app/javascript/packages/form-steps/use-history-param.ts

```diff
@@ -40,24 +14,22 @@ export function getQueryParam(queryString: string, name: string): string | null
  *
  * @see https://developer.mozilla.org/en-US/docs/Web/API/History/pushState
  *
- * @param name Parameter name to sync.
- * @param initialValue Value to use as initial in absence of another value.
- *
  * @return Tuple of current state, state setter.
  */
-function useHistoryParam(
-  name: string,
-  initialValue?: string | null,
-): [any, (nextParamValue: any) => void] {
+function useHistoryParam({ basePath }: HistoryOptions = {}): [
+  string | undefined,
+  (nextParamValue: string) => void,
+] {
   const getCurrentQueryParam = () =>
-    getQueryParam(window.location.hash.slice(1), name) ?? undefined;
```

Comment:
does this change the old flow behavior? or just the new v2 flow?

Reply:
It does change the behavior, yeah, but not in a real meaningful way. Previously, the step would be added as a hash query fragment, like `/document_capture#step=selfie`. As of these changes, it would only be the step name, `/document_capture#selfie`. Functionally, it works the same.

Also, we're not really taking advantage of the history behavior in document capture anymore, since we removed the mobile "intro" step, and selfies aren't live in production. The submission + review behavior doesn't use the history mechanic.